src/clr-base/tsconfig.base.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "declaration": true,
+    "module": "es2015",
+    "moduleResolution": "node",
+    "noFallthroughCasesInSwitch": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "../../dist/clr-base",
+    // Only necessary because @types/uglify-js can't find types for source-map
```

Comment:
I believe JSON files don't allow comments. That's why github is highlighting this as syntax error 😬

Reply:
I'll update that. Thanks!

Reply:
It isn't allowed normally, but there are some JSON parsers that allow it and consequently this is one case it can be there.

Reply:
EDIT: Jeremy is correct here. Even though Github is highlighting comments here, Typescript does allow comments in JSON format files: https://github.com/microsoft/TypeScript/issues/4987